src/docs/asciidoc/reference_doc/mail_notification.adoc

```diff
@@ -8,8 +8,14 @@
 
 = Email Notifications
 
-OpFab can send card notifications via email. These notifications are sent when a new card is published for a user, remains unread for a configured period, and the user is not connected to OpFab. The email subject includes the card title, summary, and start and end dates. The email body contains a link to the card details in OpFab.
+OpFab can send card notifications via email. These notifications are sent when a new card is published for a user, remains unread for a configured period, and the user is not connected to OpFab. The email subject includes the card title, summary, and start and end dates.
 
 == Configuring Email Notifications
 
 Users can enable email notifications in their user settings. They must provide an email address for receiving notifications and select the processes/states they want to be notified about.
+
+== Email Content
+
+The email body contains a link to the card details in OpFab. The email body can also include content from the card formatted by a template specified in the "emailBodyTemplate" field of the configuration.
+This template can use handlebars helpers for formatting but can't run javascript.
```

Comment:
"emailBodyTemplate" field in the state definition in the config.json of the corresponding process

If no template is specified , the mail is send without card detail.